### PR TITLE
feat: Improve review modal

### LINKED ISSUE #159: Improvements for review modal
image

- the username/nickname field should come before the review/text field
- the review/text field should be a large box to get people to write a detailed review

### PR DESCRIPTION
**Pull request by stormofice**
This commit introduces the following changes:
- Username input is above the review box
- Review input is now a textarea, to make it larger and to invite the user to write a longer review

There could be some styling issues with this.

Closes #159.

### CODE REVIEW

**Review by maanex — APPROVED** [review]
styling approved 👍

File: src/components/occurrence/review-modal/form/Form.module.scss [hunk 1]
```diff
@@ -100,6 +101,11 @@
     }
   }
 
+  textarea {
+    height: 100px;
+    resize: none;
+  }
+
```
Comment (GamerBene19) [inline src/components/occurrence/review-modal/form/Form.module.scss:108]: How does this size hold up on smaller display sizes?

**Review by GamerBene19 — APPROVED** [review]
Lgtm, only a small concern

File: src/components/occurrence/review-modal/form/Form.module.scss (hunk 1 above)
Comment (GamerBene19) [reply src/components/occurrence/review-modal/form/Form.module.scss:108]: Just tried it out - looks fine

Comment (stormofice) [reply src/components/occurrence/review-modal/form/Form.module.scss:108]: grafik
Looks fine in my opinion, we also use 100px for the image upload placeholder.

**Pull request merged**